react-list.es6

```diff
@@ -224,14 +224,31 @@ module.exports = class ReactList extends Component {
     this.frameRequested = window.requestAnimationFrame(this.updateFrame);
   }
 
+  waitForFrameUpdate() {
```

Comment:
@ibash this also needs an extra pair of eyes: https://github.com/superhuman/react-list/commit/571b59320d3941a5bcd0dd5bc061188b97e824be